pom.xml

```diff
@@ -42,6 +42,22 @@
                     <groupId>org.jenkins-ci.plugins</groupId>
                     <artifactId>junit</artifactId>
                 </exclusion>
+                <exclusion>
```

Comment:
Yes :( Because we need the `workflow-cps` artifact for the sample, and it brings with it lots of stuff that the enforcer complains about. It is an unholy mess to be honest.

Reply:
Ahh that explains it. Thanks!